src/plugins/data/common/field_formats/field_formats_registry.ts

```diff
@@ -180,10 +180,18 @@ export class FieldFormatsRegistry {
    * @param  {ES_FIELD_TYPES[]} esTypes
    * @return {FieldFormat}
    */
-  getDefaultInstancePlain(fieldType: KBN_FIELD_TYPES, esTypes?: ES_FIELD_TYPES[]): FieldFormat {
+  getDefaultInstancePlain(
+    fieldType: KBN_FIELD_TYPES,
+    esTypes?: ES_FIELD_TYPES[],
+    params: Record<string, any> = {}
+  ): FieldFormat {
     const conf = this.getDefaultConfig(fieldType, esTypes);
+    const instanceParams = {
+      ...conf.params,
+      ...params,
+    };
 
-    return this.getInstance(conf.id, conf.params);
+    return this.getInstance(conf.id, instanceParams);
```

Comment:
This allows timezone to be passed from the browser and formatted correctly on the server